packages/framework/tests/Unit/Model/Product/ProductVariantServiceTest.php

```diff
@@ -13,7 +16,7 @@ class ProductVariantServiceTest extends TestCase
 {
     public function testCheckProductIsNotMainVariantException()
     {
-        $productVariantService = new ProductVariantService(new ProductFactory(new EntityNameResolver([])));
+        $productVariantService = new ProductVariantService(new ProductFactory(new EntityNameResolver([]), $this->getAvailabilityFacadeMock()));
```

Comment:
Seeing how many lines you had to modify, wouldn't it make more sense to extract this into a `setUp` method?

Just assign `$this->productVariantService` and let all test methods use it.